### PR TITLE
Korean Translation

### PR DESCRIPTION
**Pull request by yor42**
## 🤔 What type of PR is this? (check all applicable)

- [ ] 🍕 Addition
- [ ] ⌨️ Productivity
- [ ] 🐛 Bug Fix
- [ ] 🔥 Optimization
- [ ] ⚙️ Configuration
- [ ] 🌟 Quality Of Life
- [x] ✨ Enhancement
- [ ] 📝 Documentation

## 📝 Description

This PR adds Korean Translation.

## 🧐 The Rationale

Not every korean are fluent with english. this will greatly help players like me who wants to play modded SMP which includes MWC with native friends.

## 🎯 Key Objectives

Add the korean Translation to MWC.

## 🚦 Testing 

1. Start minecraft
2. Set language to Korean,
3. Join world and check MWC's item names are in korean

## ⏮️ Backwards Compatibility 

This PR only adds 1 Language file. I don't think there is anything to break here 🤷‍♂️ 

## 📚 Related Issues & Documents

N/A.

## 📖 Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 Javadoc
- [ ] 🍕 Comments
- [x] 🙅 no documentation needed

## 😄 [optional] What gif best describes this PR or how it makes you feel?
cheers

### CODE REVIEW

**Comment by Paneedah** [comment]
Looks good to me. Leaving for review from @Desoroxxx

**Comment by Desoroxxx** [comment]
> Looks good to me. Leaving for review from @Desoroxxx

Do you think I speak Korean?

**Comment by Crepe2806** [comment]
> > Looks good to me. Leaving for review from @Desoroxxx
> 
> Do you think I speak Korean?

Maybe I can review this PR? I speak Korean.

**Comment by Desoroxxx** [comment]
> Maybe I can review this PR? I speak Korean.

Yeah go ahead I just want someone else that speaks Korean to review this, for all I know this could be full of slurs and insults XD

**Comment by yor42** [comment]
One thing to note: most of gun names are transliteration. and if english pronounciation is different with pronounciation from where gun originated from, I followed pronounciation of origin country.

File: src/main/resources/assets/mwc/lang/ko_KR.lang [hunk 1]
```diff
@@ -0,0 +1,1153 @@
...
+item.m82_barrett.name=M82 바렛
+item.mp40.name=MP-40
+item.m1a1_thompson.name=M1A1 톰슨
+item.m1928_thompson.name=M1928 톰슨
+item.remington870.name=레밍턴-870
+item.hk_416.name=HK 416
+item.pp91_kedr.name=PP-91 Kedr
+item.vp70.name=VP-70
+item.he_standard_combat_shotgun.name=HE-SCS
+item.cz805_bren.name=CZ-805 Bren
```
Comment (Crepe2806) [inline src/main/resources/assets/mwc/lang/ko_KR.lang:148]: 발음을 그대로 옮겨오는 방향으로 한다면 이거도
CZ-805 브렌
이렇게 번역할 수 있을듯

**Review by Crepe2806 — APPROVED** [review]
Looks fine(for now) and good to go.

File: src/main/resources/assets/mwc/lang/ko_KR.lang (hunk 1 above)
Comment (yor42) [reply src/main/resources/assets/mwc/lang/ko_KR.lang]: 반영했습니다

**Review by Crepe2806 — APPROVED** [review]

**Pull request merged**